### PR TITLE
Accessible demo

### PR DESCRIPTION
**Pull request by madCode**
Adding a demo that uses the minimum amount of code to get an Accessible Blockly toolbox and workspace on a webpage.

### CODE REVIEW

File: demos/accessible/boot.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Starts up Angular app.
+ * @author madeeha@google.com (Madeeha Ghori)
+ */
+
+document.addEventListener('DOMContentLoaded', function() {
+    ng.platform.browser.bootstrap(blocklyApp.AppView);
```
Comment (seanlip) [inline demos/accessible/boot.js]: indentation

File: demos/accessible/index.html [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+
+  <link rel="stylesheet" type="text/css" href="../../../media/accessible.css">
+</head>
+<body>
+
+  <blockly-app></blockly-app>
+
+  <script src='boot.js'></script>
+
+<xml id="blockly-toolbox-xml" style='display: none'>
```
Comment (seanlip) [inline demos/accessible/index.html]: indentation (should match 'script' tag above)

File: demos/accessible/boot.js (hunk 1 above)
Comment (seanlip) [inline demos/accessible/boot.js]: One question: is this going to be different for different external devs? If not, should it be part of core AccessibleBlockly, and just end up being one more file to be included in the list of script imports?

Comment (madCode) [inline demos/accessible/boot.js]: This file decides which Angular Component to load first. So you'll notice in the music demo, it'll be 
ng.platform.browser.bootstrap(musicGame.LevelManager);

Comment (seanlip) [inline demos/accessible/boot.js]: I see, and musicGame.LevelManager depends on the blocklyApp.AppView directive, so it loads it in turn.

This really should be documented in a README (i.e. that the dev needs to include the snippet in boot.js, or alternatively create a directive that depends on blocklyApp.AppView and bootstrap that instead). Please also provide example code snippets.

Comment (madCode) [inline demos/accessible/boot.js]: That's a good point. So I should make a README file in the demos/accessible folder?

Comment (seanlip) [inline demos/accessible/boot.js]: If it is a README explaining how to load this particular demo, then yes.

If it is a README explaining how to use AccessibleBlockly in geeral, then I think that should go in #354.

The latter is pretty much mandatory. However, you could also do an optional README for this demo if you think this is a good idea.

**Pull request merged**